### ISSUE TITLE
How to get affected_rows from connection in a transaction?

### ISSUE THREAD

**Issue by chenwei767** [blackbeam/rust-mysql-simple#211]

**Comment by chenwei767** [comment]
I think we can make some changes to `rust-mysql-simple/src/conn/transaction.rs` 

By adding a function to `Transaction`

```
impl Transaction<'_> {
....
    /// Returns number of rows affected by the last query.
    pub fn affected_rows(&self) -> u64 {
        self.conn.affected_rows()
    }
....
}
```

**Comment by blackbeam** [comment]
Hi. I believe it can be obtained easily via [`QueryResult::affected_rows`](https://docs.rs/mysql/18.0.0/mysql/struct.QueryResult.html#method.affected_rows), however the suggested addition sould be nice to have. I'll add it.

**Comment by blackbeam** [comment]
Published as [v18.1.0](https://github.com/blackbeam/rust-mysql-simple/releases/tag/v18.1.0)
